src/smart-components/order/orders-list.js

```diff
@@ -233,66 +309,47 @@ const OrdersList = () => {
-                          </Bullseye>
-                          <Title headingLevel="h1" size="lg">
-                            {meta.noData
-                              ? formatMessage(ordersMessages.noOrdersTitle)
-                              : formatMessage(filteringMessages.noResults)}
-                          </Title>
-                          <EmptyStateBody>
-                            {meta.noData
-                              ? formatMessage(
-                                  ordersMessages.noOrdersDescription
-                                )
-                              : formatMessage(
-                                  filteringMessages.noResultsDescription
-                                )}
-                          </EmptyStateBody>
-
-                          <EmptyStateSecondaryActions>
-                            {!meta.noData && (
-                              <Button
-                                variant="link"
-                                onClick={() => {
-                                  stateDispatch({
-                                    type: 'setFilteringFlag',
-                                    payload: true
-                                  });
-                                  handleFilterItems('');
-                                }}
-                              >
-                                {formatMessage(filteringMessages.clearFilters)}
-                              </Button>
-                            )}
-                          </EmptyStateSecondaryActions>
-                        </EmptyState>
-                      </Bullseye>
-                    </EmptyTable>
-                  </td>
-                </tr>
-              )}
-            </Tbody>
+          <Table
+            aria-label="orders"
+            sortBy={sortBy}
+            onSort={onSort}
+            cells={columns}
+            rows={isFetching || isFiltering ? [] : rows}
+            className="orders-table"
+          >
+            <TableHeader />
+            <TableBody />
           </Table>
+          {!(isFetching || isFiltering) && data.length === 0 && (
+            <EmptyTable>
+              <Bullseye>
+                <EmptyState>
+                  <Bullseye>
+                    <EmptyStateIcon icon={SearchIcon} />
+                  </Bullseye>
+                  <Title headingLevel="h1" size="lg">
+                    {meta.noData
+                      ? formatMessage(ordersMessages.noOrdersTitle)
+                      : formatMessage(filteringMessages.noResults)}
+                  </Title>
+                  <EmptyStateBody>
+                    {meta.noData
+                      ? formatMessage(ordersMessages.noOrdersDescription)
+                      : formatMessage(filteringMessages.noResultsDescription)}
+                  </EmptyStateBody>
+
+                  <EmptyStateSecondaryActions>
+                    {!meta.noData && (
```

Comment:
what about to level-up this condition so `EmptyStateSecondaryActions` won't be empty?